.github/workflows/format.yml

```
@@ -16,7 +16,9 @@ jobs:
       - uses: actions/setup-node@v3
```

actionlint (1.7.12)

[error] the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)